### PR TITLE
feat(TokensPermissionsPoupup): Created new popup

### LINKED ISSUE #9433: [Channels Permissions] Create token permissions required popup for channels
### Description
[Design](https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=2698%3A369970&t=iIeFeGOBx5BbbYJa-0)

Screenshot 2023-02-03 at 16 29 06

Use `JoinPermissionsOverlayPanel` component.

### PR DESCRIPTION
**Pull request by noeliaSD**
Closes #9433

### What does the PR do
- Created new dialog that shows community token permissions.
- Created storybook page for that.
- Some model properties renamed.

### Affected areas
`JoinCommunityView` and `JoinPermissionsOverlayPanel`
New popup `CommunityTokenPermissionsPopup` --> **It can only be tested in storybook.**

### Screenshot of functionality 
https://user-images.githubusercontent.com/97019400/218984768-239bc241-6272-47e3-a44d-159e3448a247.mov

### CODE REVIEW

File: ui/app/AppLayouts/Chat/popups/community/CommunityTokenPermissionsPopup.qml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Popups 0.1
+
+import AppLayouts.Chat.panels.communities 1.0
+
+StatusModal {
```
Comment (caybro) [inline ui/app/AppLayouts/Chat/popups/community/CommunityTokenPermissionsPopup.qml:10]: ... is deprecated :_) `StatusDialog` is the new cool thing :)

**Review by caybro — COMMENTED** [review]
Looks nice otherwise

File: ui/app/AppLayouts/Chat/popups/community/CommunityTokenPermissionsPopup.qml (hunk 1 above)
Comment (noeliaSD) [reply ui/app/AppLayouts/Chat/popups/community/CommunityTokenPermissionsPopup.qml]: Updated!

File: ui/app/AppLayouts/Chat/popups/community/CommunityTokenPermissionsPopup.qml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+    QtObject {
+        id: d
+
+        readonly property int maxWidth: 640
+        readonly property int minWidth: 300
+        readonly property int maxHeight: 480
+
+        function getHorizontalPaddings() {
+            return root.leftPadding + root.rightPadding
```
Comment (caybro) [inline ui/app/AppLayouts/Chat/popups/community/CommunityTokenPermissionsPopup.qml:26]: This is just `root.horizontalPadding` already (a builtin)

File: ui/app/AppLayouts/Chat/panels/communities/JoinPermissionsOverlayPanel.qml [hunk 3]
```diff
@@ -22,11 +22,12 @@ Control {
...
-    property var communityHoldings
+    property var communityHoldingsModel
     property string channelName
-    property var viewOnlyHoldings
-    property var viewAndPostHoldings
-    property var moderateHoldings
+    property var viewOnlyHoldingsModel
+    property var viewAndPostHoldingsModel
+    property var moderateHoldingsModel
+    property bool isOnlyPanel: false
```
[the remark this answers is not in the document]
Comment (noeliaSD) [reply ui/app/AppLayouts/Chat/panels/communities/JoinPermissionsOverlayPanel.qml:30]: This component contains 4 different holding panels + a button + a text. So this property will be used to hide these button and text components and only show the panels. That's why the property name is `isOnlyPanel` but I can agree the name can be confusing so happy to change it!! Just propose me a better name, please! :) :pray: 

Maybe... `showOnlyPanels`??

File: ui/app/AppLayouts/Chat/popups/community/CommunityTokenPermissionsPopup.qml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        ScrollBar.horizontal.policy: ScrollBar.AsNeeded
+        contentHeight: overlayPannel.implicitHeight
+        contentWidth: overlayPannel.implicitWidth
+        padding: 0
+
+        Item {
+            width: Math.max(root.width - d.getHorizontalPaddings(), overlayPannel.implicitWidth)
+            height: Math.max(root.height - d.getVerticalPaddings(), overlayPannel.implicitHeight)
+
+            JoinPermissionsOverlayPanel {
```
[the remark this answers is not in the document]
Comment (noeliaSD) [reply ui/app/AppLayouts/Chat/popups/community/CommunityTokenPermissionsPopup.qml:53]: True! I will remove the intermediate Item!

**Review by alexandraB99 — APPROVED** [review]
lgtm!

**Review by caybro — APPROVED** [review]
Nice!

**Review by alexjba — APPROVED** [review]
LGTM! 👍

**Review by glitchminer — APPROVED** [review]
Tested, approved

**Pull request merged**